bin/yarn.ps1

```diff
@@ -0,0 +1,6 @@
+function Get-ScriptDirectory {
+	$Invocation = (Get-Variable MyInvocation -Scope 1).Value
+	Split-Path $Invocation.MyCommand.Path
+}
```

Comment:
Is there any difference between the output of this function and $PSScriptRoot?

Reply:
`$PSScriptRoot` may not have been considered because it was introduced in PowerShell 3. It would not work for users of PowerShell <3.

Reply:
As the author `$PSScriptRoot` wasn't considered because I just learnt about it from @felixfbecker. Want me to update the PR?

Reply:
If `$PSScriptRoot` was introduced in PS 3 that would mean it is not supported in Windows 7 - I don't know what the support policy is here from yarn / NodeJS?